.travis.yml

```diff
@@ -5,6 +5,7 @@ services:
   - postgresql # starts up postgres
 addons:
   postgresql: '9.6' # highest available postgres version on Travis
+  chrome: stable
```

Comment:
Travis needs to install a web browser for the `karma` test runner. ( Karma runs tests in an actual web browser, very realistic. )